evaluate/metrics/metrics.go

```diff
@@ -9,83 +9,61 @@ import (
 
 	pkgerrors "github.com/pkg/errors"
 	"golang.org/x/exp/maps"
-	"gonum.org/v1/gonum/stat"
 )
 
 // Metrics holds numerical benchmarking metrics.
-// TODO Move all metrics to assessment. https://github.com/symflower/eval-dev-quality/issues/34
 type Metrics struct {
-	// Executed is the number of benchmarking candidates with successful execution.
-	Executed uint
-	// Problems is the number of benchmarking candidates with problems.
-	Problems uint
 	// Total is the total number of benchmarking candidates.
 	Total uint
```

Comment:
Let's drop the total

Reply:
https://github.com/symflower/eval-dev-quality/pull/38/commits/943cf127daff7f88212d1c50d058a5a16d5c50cf#r1568537097

Reply:
Not sure where this links to other than the commit :thinking:

Reply:
But for some reason, the link of a conversion depends on the view. w t h...